.devcontainer/Dockerfile

```diff
@@ -1,13 +1,23 @@
-FROM maven:3.6.3-jdk-11
+FROM maven:3.6.3-jdk-11-slim
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update
-RUN apt-get -y install --no-install-recommends apt-utils dialog dnsutils httpie bash-completion curl wget git unzip
-RUN apt-get -y install git procps lsb-release
+#Install Common Tools
+RUN apt-get update && apt-get -y install --no-install-recommends apt-transport-https apt-utils bash-completion curl dialog dnsutils git httpie lsb-release procps unzip wget
+
+#Install Azure CLI
 RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
 
 # Link Java Installation to defined /docker-java-home
 RUN ln -s "${JAVA_HOME}" /docker-java-home
 
-ENV DEBIAN_FRONTEND=
\ No newline at end of file
+#Install .NET Core SDK
+RUN wget https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb -O /tmp/packages-microsoft-prod.deb && \
+ dpkg -i /tmp/packages-microsoft-prod.deb && apt-get update && apt-get -y install --no-install-recommends dotnet-sdk-3.1
```

Comment:
Is it possible to do the wget first so you can combine this `apt-get install` for dotnet-sdk-3.1 with the apt-get install in line 6?

Reply:
Good observation, but unfortunately that wouldn't work because wget wouldn't be installed at that point (wget gets installed in line 6), but curl is available :) Thanks for the feedback

Reply:
Gotcha that makes sense.